### PR TITLE
fix: broken links in HTTP docs

### PR DESCRIPTION
**Pull request by bsmth**
### Description

This PR fixes the following cases in prose:

* ` www.` renders as a link. Changing to ` 'www'.`
* `www.netlify.com` renders as a link. Changing to `netlify.com`

__Other fixes:__

* Correct broken internal xrefs
* Remove or fix broken external links

### CODE REVIEW

File: files/en-us/web/http/browser_detection_using_the_user_agent/index.md
Comment (bsmth) [inline files/en-us/web/http/browser_detection_using_the_user_agent/index.md:1]: This whole page needs a large revision. For now, I'd like to limit to removing broken links.

File: files/en-us/web/http/status/102/index.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ This status code is only sent if the server expects the request to take signific
 
 {{Specifications}}
 
+## Browser compatibility
```
Comment (bsmth) [inline files/en-us/web/http/status/102/index.md:26]: Added so the deprecated header points to an existing location on the page.

**Review by teoli2003 — APPROVED** [review]
👍 Sounds good. Thank you!

**Comment by bsmth** [comment]
Thank you!

**Pull request merged**